src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
+import java.util.Random;
 
 public class ColorSupplier {
+    Random random = new Random();
     public String getRandomColor() {
-        return null;
+        int index = new Random().nextInt(Color.values().length);
+        Color color = Color.values()[index];
+        return color.toString();
```

Comment:
According to the checklist, you should use 'name()' instead of 'toString()' to get the String representation of enum constants to avoid potential issues with overridden 'toString()' methods.